provider/local/converge.go

```diff
@@ -388,6 +392,20 @@ func (p *Provider) serviceContainers(services manifest.Services, app, release st
 			e[k] = v
 		}
 
+		// add links
+		for _, sl := range s.Links {
+			for _, s := range m.Services {
+				if s.Name == sl {
+					u, err := p.serviceURL(app, s.Name)
+					if err != nil {
+						return nil, err
+					}
+
+					e[fmt.Sprintf("%s_URL", strings.ToUpper(sl))] = u
```

Comment:
Seems like this will break on services with dashes in the name, e.g. `my-service`

Reply:
oh yeah. good call!

Reply:
and it seems to be on par with the aws provider 

https://github.com/convox/rack/blob/master/provider/aws/formation/app.json.tmpl#L421

https://github.com/convox/rack/blob/master/provider/aws/template.go#L103

perhaps they should change to use upper / upperName instead?

Reply:
or no nvm...that seems to be "the other way around"

Reply:
Looks like you're right, go ahead and ignore this for now, will have to fix globally